src/OpenIdConnect.php

```diff
@@ -75,6 +76,20 @@
  */
 class OpenIdConnect extends OAuth2
 {
+    // https://openid.net/specs/openid-connect-core-1_0.html#rfc.section.2
```

Comment:
That should be in a docblock that also contains `@since 2.2.12`.

Reply:
Updated docblock